### ISSUE TITLE
Naming issues for local .threepio

### ISSUE THREAD

**Issue by prichey** [40Digits/threepio#3]
I created a local `.threepio` file and `threepio --help` showed my config settings as they had been defined in that file, but when I went to run `threepio` I got an error, `'threepio.json' not found. Please create one or run`threepio --help`to see available tasks to run.`. I renamed `.threepio` to `threepio.json` and reran; I didn't get the error message about needing a json file but the settings I had defined in that file weren't pulled in and it reverted to the global defaults.

**Comment by grantholle** [comment]
So your `.threepio` file is config, which you had right initially.

A `threepio.json` file is a workflow file, which is a JSON array of the task names you want to run. By simply running `threepio` it expects there to be a workflow file (`threepio.json`). Perhaps the message could be more indicative of what it expects for a workflow file.

**Comment by prichey** [comment]
Ah, that makes more sense. Any reason not to include the workflows array in the local `.threepio` as just another value (`workflow: ['mysql', ...]`)? That way you could define a workflow either locally or globally, which seems advantageous over having to have (potentially) duplicate workflow files across different projects. Or maybe you could have different named workflows globally (`workflows: {'wp': [], 'node': []},`), then run it with `threepio wp`? Not sure that use case would come up often though, since the written tasks mostly focus on WP specifically. Either way, a more descriptive message would have tipped me off that it wasn't a bug and `threepio.json` was necessary to run the workflow.

**Comment by grantholle** [comment]
I updated the [error message](https://github.com/40Digits/threepio/commit/d6ae6eda02343926f4cad573584fc69ea6444323).

I wanted to keep them separate because I saw one being configuration and workflows not so much as configuration, but more of giving instructions on what to do, which could be included in a repo, where configuration wouldn't.

I do like the idea of defining a workflows object with different "profiles." Good idea.
